intertidal/utils.py

```diff
@@ -143,14 +143,14 @@ def intertidal_hillshade(
 ):
     """
     Create a hillshade array for an intertidal zone given an elevation
-    array and an extents array.
+    array and a frequency array.
 
     Parameters
     ----------
     elevation : str or xr.DataArray
-        An xr.DataArray or a path to the elevation raster file.
```

Comment:
Did you mean to delete this line (old version: line 151) from the elevation parameter documentation?